packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -299,6 +307,21 @@ abstract class ServerlessClusterBase extends Resource implements IServerlessClus
     });
   }
 
+  /**
+   * Grant the given identity to access to the Data API
+   *
+   * @param grantee The principal to grant access to
+   */
+  public grantDataApi(grantee: iam.IGrantable): iam.Grant {
```

Comment:
I feel like there's maybe a word missing here. What do you think about `grantUseDataApi` or `grantDataApiAccess` or something similar to that?

Reply:
`grantUseDataApi`, `grantDataApiAccess`, `grantInvokeDataApi`, or even something like `grantDataService`. I **think** the service's official name is `RDS Data Service` (see [here](https://docs.aws.amazon.com/rdsdataservice/latest/APIReference/Welcome.html)) and it's API is aptly called `RDS Data Service API` (Data API for short?).

Reply:
It's already called `Data API` in several other places in the code, so we should probably stick to it

Reply:
I like both `grantUseDataApi` and `grantDataApiAccess`. I'll leave the final decision to @njlynch 🙂.